bobtemplates/django/+package.fullname+/README.md.bob

```diff
@@ -0,0 +1,124 @@
+# {{{package.fullname}}}
+
+## Development
+
+### Setup
+
+A few steps are needed to setup the backend.
+Run these steps within the project folder.
+
+1. Create and activate a python virtual environment:
+    `python3 -m venv . && source bin/activate`
+
+2. Install the application and its requirements:
+    `pip install -e .[development]`
```

Comment:
There is no such extras `[development]`, it's name is `[dev]`. So this will lead to:
```python
Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/jone/projects/packages/bobtemplates.4teamwork/generated/moneypenny/lib/python3.6/site-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
    utility.execute()
  File "/Users/jone/projects/packages/bobtemplates.4teamwork/generated/moneypenny/lib/python3.6/site-packages/django/core/management/__init__.py", line 347, in execute
    django.setup()
  File "/Users/jone/projects/packages/bobtemplates.4teamwork/generated/moneypenny/lib/python3.6/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/jone/projects/packages/bobtemplates.4teamwork/generated/moneypenny/lib/python3.6/site-packages/django/apps/registry.py", line 89, in populate
    app_config = AppConfig.create(entry)
  File "/Users/jone/projects/packages/bobtemplates.4teamwork/generated/moneypenny/lib/python3.6/site-packages/django/apps/config.py", line 90, in create
    module = import_module(entry)
  File "/usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 948, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'rest_framework_swagger'
```